raiden/tests/unit/transfer/mediated_transfer/test_initiatorstate.py

```diff
@@ -323,7 +323,7 @@ def test_state_wait_secretrequest_invalid_amount():
         block_number,
     )
 
-    assert len(iteration2.events) == 0
+    assert not iteration2.events
```

Comment:
Honestly I only changed this because the linter was complaining. Usually the `not var` is considered more readable, but yeah, it can be a source of bugs. Do you want me to revert it?

Reply:
I would prefer it yes. And perhaps we should also write it in the contributing guide so that we remember it from now on.

Which linter? Pylint?

Reply:
> Which linter? Pylint?

I don't know, it's integrated in my editor (could be flake8, pylint, any of the flake8 extensions, and a few others)